### PR TITLE
Updated to reflect cvss scores being added to output table.

### PR DESCRIPTION
**Pull request by hayleycd**
Addresses #418

I added CVSS to the output page. I calculated the scores using [this](https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H). I think it is doing the same thing as we are, but I have a note to confirm with our tool once it is released. 

I added a section on the usage page that lists everything that is outputted, so I can say that CVSS is calculated using the [severity[].score](https://ossf.github.io/osv-schema/#severity-field) field. I am open to other ideas for how to present that information though. 

[Preview available here](https://hayleycd.github.io/osv-scanner/)

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ osv-scanner --format table your/project/dir
...
-├─────────────────────────────────────┼───────────┼──────────────────────────┼─────────┼────────────────────┤
-│ https://osv.dev/GHSA-c3h9-896r-86jm │ Go        │ github.com/gogo/protobuf │ 1.3.1   │ path/to/go.mod     │
-│ https://osv.dev/GHSA-m5pq-gvj9-9vr8 │ crates.io │ regex                    │ 1.3.1   │ path/to/Cargo.lock │
-╰─────────────────────────────────────┴───────────┴──────────────────────────┴─────────┴────────────────────╯
+╭──────────────────────────────────────┬─────────────┬───────────┬──────────────────────────┬─────────┬────────────────────╮
+│ OSV URL                              │    CVSS     │ ECOSYSTEM │ PACKAGE                  │ VERSION │ SOURCE             │
+├──────────────────────────────────────┼─────────────┼───────────┼──────────────────────────┼─────────┼────────────────────┤
+│ https://osv.dev/GHSA-c3h9-896r-86jm  |    8.6      │ Go        │ github.com/gogo/protobuf │ 1.3.1   │ path/to/go.mod     │
+│ https://osv.dev/GHSA-m5pq-gvj9-9vr8  |    7.5      │ crates.io │ regex                    │ 1.3.1   │ path/to/Cargo.lock │
+╰──────────────────────────────────────┴─────────────┴───────────┴──────────────────────────┴─────────┴────────────────────╯
```
Comment (another-rex) [inline docs/output.md:28]:
```suggestion
╭──────────────────────────────────────┬─────────────┬───────────┬──────────────────────────┬─────────┬────────────────────╮
│ OSV URL                              │ CVSS │ ECOSYSTEM │ PACKAGE                  │ VERSION │ SOURCE             │
├──────────────────────────────────────┼─────────────┼───────────┼──────────────────────────┼─────────┼────────────────────┤
│ https://osv.dev/GHSA-c3h9-896r-86jm  | 8.6  │ Go        │ github.com/gogo/protobuf │ 1.3.1   │ path/to/go.mod     │
│ https://osv.dev/GHSA-m5pq-gvj9-9vr8  | 7.5  │ crates.io │ regex                    │ 1.3.1   │ path/to/Cargo.lock │
╰──────────────────────────────────────┴─────────────┴───────────┴──────────────────────────┴─────────┴────────────────────╯
```

**Comment by hayleycd** [comment]
It took a little bit of tweaking for me to get the table looking correct. This is the closest to correct that it has been. 

[Here](https://raw.githubusercontent.com/hayleycd/image-library/main/osv-docs/table.png) is what it currently looks like on the github page (not running locally, but deployed from my branch).

File: docs/output.md [hunk 2]
```diff
@@ -4,11 +4,33 @@ title: Output
...
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+## Explanation of output data
+
+For every vulnerability found, OSV-Scanner will display the following information:
+
+- OSV URL: Link to the database entry for the vulnerability
```
Comment (another-rex) [inline docs/output.md:23]:
```suggestion
- OSV URL: Link to the osv.dev entry for the vulnerability
```

**Review by another-rex — APPROVED** [review]
LGTM! Thanks for putting the explanation in, just one comment.

**Pull request merged**